src/routes/wohnen/components/Offer.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+
+import React from 'react'
+import WohnenOfferModel from 'modules/endpoint/models/WohnenOfferModel'
+import WohnenFormData from 'modules/endpoint/models/WohnenFormData'
+import styled from 'styled-components'
+import type { AccommodationType } from '../../../modules/endpoint/models/WohnenFormData'
+import type { TFunction } from 'react-i18next'
+import { translate } from 'react-i18next'
+import ListElement from '../../../modules/common/components/ListElement'
+
+type PropsType = {
+  offer: WohnenOfferModel<*>,
+  t: TFunction
+}
+
+const Header = styled.div`
+  padding: 15px 5px 5px;
+  font-weight: 700;
+`
+
+const RowTitle = styled.div`
+  flex: 50%;
+`
+
+const RowValue = styled.div`
+  flex: 50%;
+`
+
+const Row = styled.div`
+  display: flex;
+`
+
+class Offer extends React.Component<PropsType> {
```

Comment:
Maybe we shouldn't call this file/component Offer... sounds like a model (we have this problem a lot^^). Maybe sth like OfferDetailView would be better (but not really because of the 'View' suffix^^)

Reply:
Renamed to OfferDetail